ci/serverless/dlq_rspec_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -ex
+
+vault_path=secret/ci/elastic-logstash/serverless-test
+# Since we are using the system jruby, we need to make sure our jvm process
```

Comment:
It's not system ruby, right? we're using the one we put under `vendor/` during the gradle tasks